### PR TITLE
Documentation -- Function Headers -- sys_prc77

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Handle documentation issue #107: sys_prc77 component. WIP.

### CODE REVIEW

**Review by jonpas — CHANGES_REQUESTED** [review]
No semi-colons in header examples.

File: addons/sys_prc77/functions/fnc_onFunctionKnobPress.sqf [hunk 1]
```diff
@@ -1,37 +1,38 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Handles the event of turning the function radio knob and selects the radio mode: off (0), on (1),
+ * squelch (2), retrans (3) and lite (4).
```
Comment (jonpas) [inline addons/sys_prc77/functions/fnc_onFunctionKnobPress.sqf]: All 5 modes on same line is cleaner to read.

File: addons/sys_prc77/functions/fnc_onPresetKnobPress.sqf [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    _fnc = 0;
+};
+
+// Select Presetarray
+// Get the full Preset array [[x,y],[u,v]]
+private _currentPresets = GET_STATE("currentPreset");
+
+
+if (_fnc == 0) then {
+    //Select array according to preset handler (left,right) [x,y]
```
Comment (jonpas) [inline addons/sys_prc77/functions/fnc_onPresetKnobPress.sqf]: Spaces after `//` here too.

File: addons/sys_prc77/radio/fnc_getExternalAudioPosition.sqf [hunk 3]
```diff
@@ -1,16 +1,20 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Only used if a radio has an internal speaker. Since the AN/PRC 77 has none, this function returns an
+ * array of zeros and does nothing.
```
Comment (jonpas) [inline addons/sys_prc77/radio/fnc_getExternalAudioPosition.sqf:4]: `zeroes`

File: addons/sys_prc77/radio/fnc_getListInfo.sqf [hunk 4]
```diff
@@ -1,16 +1,20 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * For a AN/PRC 77 this function is the same as acre_sys_prc77_getChannelDescription,
```
Comment (jonpas) [inline addons/sys_prc77/radio/fnc_getListInfo.sqf]: `same as FUNC(getChannelDescription)` or just `same as getChannelDescription`.

File: addons/sys_prc77/radio/fnc_getState.sqf [hunk 5]
```diff
@@ -1,21 +1,25 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Function to obtain the content of the radio data hash for the event data key
```
Comment (jonpas) [inline addons/sys_prc77/radio/fnc_getState.sqf]: Missing final dot.

File: addons/sys_prc77/radio/fnc_handleBeginTransmission.sqf [hunk 6]
```diff
@@ -1,21 +1,22 @@
...
- * [ARGUMENTS] call acre_COMPONENT_fnc_FUNCTIONNAME
+ * ["ACRE_PRC77_ID_1", "handleBeginTransmission", [], [], false] call acre_sys_prc77_fnc_handleBeginTransmission
  *
  * Public: No
  */
 #include "script_component.hpp"
 
-params ["_radioId", "_eventKind", "_eventData"];
-
 true;
```
Comment (jonpas) [inline addons/sys_prc77/radio/fnc_handleBeginTransmission.sqf]: Should have no semi-colon.

File: addons/sys_prc77/radio/fnc_openGui.sqf [hunk 7]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Author: ACRE2Team
+ * Opens the GUI of the radio. This is trigered by either double-click in the inventory,
+ * through the ACE interact menu (if available) or through the keybinding "open radio".
+ *
+ *
```
Comment (jonpas) [inline addons/sys_prc77/radio/fnc_openGui.sqf]: One line less.

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/sys_prc77/radio/fnc_getExternalAudioPosition.sqf (hunk 3 above)
Comment (TheMagnetar) [reply addons/sys_prc77/radio/fnc_getExternalAudioPosition.sqf:4]: Array of zeros is correct.

Comment (jonpas) [reply addons/sys_prc77/radio/fnc_getExternalAudioPosition.sqf:4]: Apparently both are, urgh.

**Review by jonpas — APPROVED** [review]

**Pull request merged**